integrations/langchain/tests/test_toolkit.py

```diff
@@ -159,3 +164,34 @@ def test_uc_function_to_langchain_tool():
         )
         assert tool.name == get_tool_name(f"{CATALOG}.{SCHEMA}.test")
         assert json.loads(tool.func(x="some_string"))["value"] == "some_string"
+
+
+@requires_databricks
+@pytest.mark.parametrize("use_serverless", [True, False])
+def test_langgraph_agents(monkeypatch, use_serverless):
+    monkeypatch.setenv(USE_SERVERLESS, str(use_serverless))
+    client = get_client()
+    with create_function_and_cleanup(client, schema=SCHEMA) as func_obj:
+        toolkit = UCFunctionToolkit(function_names=[func_obj.full_function_name], client=client)
+        system_message = "You are a helpful assistant. Make sure to use tool for information."
+        llm = ChatDatabricks(endpoint="databricks-meta-llama-3-1-70b-instruct")
+        agent = create_react_agent(llm, toolkit.tools, state_modifier=system_message)
+        chain = agent | RunnableGenerator(wrap_output)
+        # we need this mock as the serving endpoint is not stable
```

Comment:
Are all of the FMAPI endpoints unstable on our workspace or just large models like the 70b variant?

Reply:
I tried 70b and 405b, they're not generating correct results. But OpenAI works fine, so I just mock the response lol